storage/src/test/java/tech/pegasys/artemis/storage/client/RecentChainDataTest.java

```diff
@@ -189,7 +189,33 @@ public void getStateBySlot_returnGenesisBlockWhenItIsTheBestState() {
     when(store.getBlockState(GENESIS_BLOCK_ROOT)).thenReturn(newState);
     when(store.getBlock(GENESIS_BLOCK_ROOT)).thenReturn(GENESIS_BLOCK);
 
-    assertThat(storageClient.getBlockBySlot(UnsignedLong.ZERO)).contains(GENESIS_BLOCK);
+    assertThat(storageClient.getStateInEffectAtSlot(UnsignedLong.ZERO)).contains(newState);
+  }
+
+  @Test
+  public void getStateInEffectAtSlot_returnStateFromLastBlockWhenSlotsAreEmpty() {
```

Comment:
It's unclear to me what's the difference between the two tests here and specifically what they are testing. I've removed both of their bestState and newState lines, kept only a random beacon state generation instead, and that resulted in both tests passing as well.